lochness/tree/__init__.py

```diff
@@ -117,18 +117,18 @@ def get(data_type, base, **kwargs):
             processed_folder = Templates[data_type]['processed'].substitute(
                     base=base, **kwargs)
 
-    if kwargs.get('makedirs', True):
-        if raw_folder and not os.path.exists(raw_folder):
-            logger.debug(f'creating raw folder {raw_folder}')
-            os.makedirs(raw_folder)
-        if processed_folder and not os.path.exists(processed_folder):
+    if kwargs.get('processed', True):
```

Comment:
https://github.com/PREDICT-DPACC/lochness/pull/27#issuecomment-863456550

Now `tree.get` only creates either `raw` or `processed` directory, depending on the `processed` option.

Reply:
This is the outcome of the latest clone:

```
(base) [tb571@rc-predict tb571_lochness_test]$ tree MF-PHOENIX/GENERAL/BWH
MF-PHOENIX/GENERAL/BWH
├── BWH_metadata.csv
├── processed
│   └── phone
│       ├── sub01234
│       ├── sub01235
│       └── sub01236
└── raw
    └── phone
        ├── sub01234
        │   ├── BLS-F6VVM-phone_accel_activityScores_hourly-day1to43.csv
        │   ├── BLS-F6VVM-phone_accel_activityScores_hourly-day1to51.csv
        │   └── BLS-F6VVM-phone_accel_activityScores_hourly-day1to79.csv
        ├── sub01235
        │   ├── BLS-F6VVM-phone_accel_activityScores_hourly-day1to43.csv
        │   ├── BLS-F6VVM-phone_accel_activityScores_hourly-day1to51.csv
        │   └── BLS-F6VVM-phone_accel_activityScores_hourly-day1to79.csv
        └── sub01236
            ├── BLS-F6VVM-phone_accel_activityScores_hourly-day1to43.csv
            ├── BLS-F6VVM-phone_accel_activityScores_hourly-day1to51.csv
            └── BLS-F6VVM-phone_accel_activityScores_hourly-day1to79.csv

10 directories, 10 files

```